db/migrate/20190426200757_create_reviews.rb

```diff
@@ -1,13 +0,0 @@
-class CreateReviews < ActiveRecord::Migration[5.2]
```

Comment:
Why was this file deleted?

Reply:
we had 2 migrations when we merged for some reason so we had to delete one.

Reply:
nvm I see what you mean :D